packages/plugins/rubygem-foreman_dhcp_browser/rubygem-foreman_dhcp_browser.spec

```diff
@@ -1,36 +1,27 @@
 # template: foreman_plugin
-%{?scl:%scl_package rubygem-%{gem_name}}
-%{!?scl:%global pkg_name %{name}}
-
 %global gem_name foreman_dhcp_browser
 %global plugin_name dhcp_browser
-%global foreman_min_version 1.12.0
+%global foreman_min_version 3.14.0
```

Comment:
well, no?

Reply:
Argh, in https://github.com/theforeman/foreman_dhcp_browser/commit/13ce09e0522eda83bbeca08f12b6ad54ea8d65c1 I assumed Rails 7 (which is 3.14) but it is probably also compatible with 3.13. I think this requires another release :(

Reply:
https://github.com/theforeman/foreman_dhcp_browser/pull/29